### ISSUE TITLE
implement support for structured replies in both nbd-client and nbd-server

### ISSUE THREAD

**Issue by nhooyr** [NetworkBlockDevice/nbd#86]
Neither support structured replies (nbd-client doesn't support sending the option to negotiate structured replies)

**Comment by yoe** [comment]
Yes, I know it hasn't been implemented yet. This will happen when I find the time, at least for the server side.

For the client side, kernel support will be needed.

**Comment by nhooyr** [comment]
Going to close as it has been a long time and I'm not sure if this is relevant anymore.

**Comment by yoe** [comment]
It's definitely still relevant. I've been looking at working on this, too.

**Comment by yoe** [comment]
Server side has been implemented as of 3.25.

**Comment by nhooyr** [comment]
This is my oldest filed issue on which there's been meaningful progress. Nice @yoe

**Comment by rrauch** [comment]
I'm interested in client support too. Would really help with my use-case.
